modules/network-observability-flowcollector-view.adoc

```diff
@@ -57,14 +57,15 @@ spec:
         type: configmap
         name: loki-gateway-ca-bundle
         certFile: service-ca.crt
+        namespace: loki-namespace            <5>
```

Comment:
```suggestion
        # namespace: loki-namespace            <5>
```

Let's comment that by default so the users can still install loki in `netobserv` namespace following the doc and will understand they can optionnaly customize it

Reply:
Ok yeah that's a good suggestion. I was also thinking, all the examples use the `netobserv` namespace. So I'll make it more clear that different namespace is an option but in our examples we use the same `netobserv` namespace across. And I'll comment that part out of the CRD too, good idea. TY!